hyperon_das_atomdb/adapters/redis_mongo_db.py

```diff
@@ -87,31 +94,68 @@ class MongoDBIndex(Index):
     def __init__(self, collection: Collection) -> None:
         self.collection = collection
 
-    def create(self, atom_type: str, field: str, **kwargs) -> Tuple[str, Any]:
+    def create(
+        self,
+        atom_type: str,
+        field: str,
+        fields: Optional[List[str]] = None,
+        index_type: Optional[MongoIndexType] = None,
+        **kwargs,
+    ) -> Tuple[str, Any]:
         conditionals = {}
 
         for key, value in kwargs.items():
             conditionals = {key: {"$eq": value}}
             break  # only one key-value pair
```

Comment:
I know this was here previously to your change, but I think it's worth it to refactor it.

```suggestion
        key, value = next(iter(kwargs.items())):
        conditionals = {key: {"$eq": value}}
```

Reply:
I'll add an IF because kwargs can be empty